debbuild

```diff
@@ -750,7 +750,8 @@ ENDIF: if (/^\s*%endif/) {
       if (my ($dname,$dvalue) = /^(Summary|Group|Version|Conflicts|Provides|
           BuildArch(?:itecture)?|BuildRequires|BuildConflicts|
           Recommends|Supplements|Suggests|Enhances|Obsoletes|Breaks|Replaces|
-          PreReq|Requires(?:\(pre(?:un)?\))?|Pre-Depends):\s+(.+)$/ix) {
+          PreReq|Requires(?:\((?:pre|post)(?:un)?\))?|Pre-Depends):\s+(.+)$/ix) {
```

Comment:
I've run a bunch of cases through a regex tester, and this looks solid. Good work.